### PR TITLE
Conditionally skip iam and cert policy tests

### PR DESCRIPTION
**Pull request by zyjjay**
ref: https://issues.redhat.com/browse/ACM-4811

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -277,6 +279,9 @@ MANAGED_CLUSTER_NAMESPACE ?= $(MANAGED_CLUSTER_NAME)
 
 .PHONY: e2e-test
 e2e-test: e2e-dependencies
+	if [ "$(UPSTREAM_TEST)" = "true" ]; then\
```
Comment (JustinKuli) [inline Makefile:282]: I usually reference this styleguide for bash: https://google.github.io/styleguide/shellguide.html#s6.3-tests, which would suggest
```suggestion
	if [[ "$(UPSTREAM_TEST)" == "true" ]]; then\
```
But I think what you have here will work... And it looks like the single bracket syntax is used somewhere else in this file, so it's up to you if you want to change it in all the places or keep it like this.

**Review by JustinKuli — APPROVED** [review]
/hold for optional suggestion

Easiest way to test this is probably to merge it and just try :)

File: Makefile (hunk 1 above)
Comment (dhaiducek) [reply Makefile:282]: @JustinKuli The default shell is `sh` for Make, so double brackets won't work since we're not setting the shell otherwise.

Comment (JustinKuli) [reply Makefile:282]: Oh, wow, good to know

**Comment by zyjjay** [comment]
Going ahead and merging this!

**Pull request merged**